docs/tutorials/basic/csharp.md

```diff
@@ -166,13 +167,13 @@ To generate the code, the following command should be run from the
 - Windows
 
   ```
-  > packages\Grpc.Tools.0.14.0\tools\windows_x86\protoc.exe -I../../protos --csharp_out RouteGuide --grpc_out RouteGuide ../../protos/route_guide.proto --plugin=protoc-gen-grpc=packages\Grpc.Tools.0.14.0\tools\windows_x86\grpc_csharp_plugin.exe
+  > packages\Grpc.Tools.1.0.0\tools\windows_x86\protoc.exe -I../../protos --csharp_out RouteGuide --grpc_out RouteGuide ../../protos/route_guide.proto --plugin=protoc-gen-grpc=packages\Grpc.Tools.1.0.0\tools\windows_x86\grpc_csharp_plugin.exe
```

Comment:
This should probably become a variable reference, like we do with the git clone.